src/app/api/v1/user/email/route.ts

```diff
@@ -110,6 +111,6 @@ export async function POST(req: NextRequest) {
     }
   } else {
     // Not Signed in, redirect to home
-    return NextResponse.redirect(process.env.SERVER_URL ?? "/", 301);
+    return NextResponse.redirect(config.serverUrl, 301);
```

Comment:
nit: Same qn as above about omitting the nullish operator. Then again... if the server url wasn't defined we'd be running into bigger issues 😅